pkg/apis/keycloak/v1alpha1/keycloak_types.go

```diff
@@ -181,7 +181,8 @@ type KeycloakExternalAccess struct {
 type KeycloakExternalDatabase struct {
 	// If set to true, the Operator will use an external database.
 	// pointing to Keycloak.
-	Enabled bool `json:"enabled,omitempty"`
+	// +kubebuilder:default:=true
+	Enabled bool `json:"enabled"`
```

Comment:
I'm afraid we can't make this breaking change. Making it  required (non optional) would break OLM upgrades, AFAIK. At the same time, changing it's default value would be a problem for existing installations. We do not have an automatic migration path from embedded DB to external DB.